### PR TITLE
Add Setter for Ragdoll

### PR DESCRIPTION
**Pull request by louis1706**

### CODE REVIEW

**Comment by Thundermaker300** [comment]
Tested?

**Comment by louis1706** [comment]
> Tested?

Nope

File: Exiled.API/Features/Ragdoll.cs [hunk 1]
```diff
@@ -152,12 +174,17 @@ public DateTime CreationTime
...
-        public RoleTypeId Role => NetworkInfo.RoleType;
+        public RoleTypeId Role
+        {
+            get => NetworkInfo.RoleType;
+            set => NetworkInfo = new(NetworkInfo.OwnerHub, NetworkInfo.Handler, value, NetworkInfo.StartPosition, NetworkInfo.StartRotation, NetworkInfo.Nickname, NetworkInfo.CreationTime);
+        }
 
         /// <summary>
-        /// Gets a value indicating whether or not the ragdoll has expired and SCP-049 is unable to revive it.
+        /// Gets a value indicating whether or not the ragdoll has expired and SCP-049 is unable to revive it if was not being targets.
```
Comment (Thundermaker300) [inline Exiled.API/Features/Ragdoll.cs:186]: what

Comment (louis1706) [reply Exiled.API/Features/Ragdoll.cs:186]: it's because Scp049 is able to revive if it's was a previous target of Scp049 (with it's SenseAbility)

**Review by joker-119 — APPROVED** [review]

**Review by Misaka-ZeroTwo — APPROVED** [review]

**Pull request merged**